draft-ietf-v6ops-464xlaton.xml

```diff
@@ -245,7 +245,7 @@ Despite the word "wireless" in the name, this architecure is applicable for wire
 </li>
 </ul>
 <t>
-In the single-address model, the CLAT instance needs a single IPv6 CLAT address (see <xref target="v6addr"/>) and a single IPv4 address only (distinct from the one or more IPv6 addresses used by the node running CLAT for its own native IPv6 connectivity).
+In the single-address model, the CLAT instance needs a single IPv4 CLAT address and a single IPv6 CLAT address only (that IPv6 address is distinct from the one or more IPv6 addresses used by the node running CLAT for its own native IPv6 connectivity, see <xref target="v6addr"/>).
```

Comment:
```suggestion
In the single-address model, the CLAT instance needs a single IPv4 CLAT address and a single CLAT-only IPv6 address (which is distinct from the one or more IPv6 addresses used by the node running CLAT for its own native IPv6 connectivity, see <xref target="v6addr"/>).
```

Non-blocking, just thought this would be more clear